services/catarse/lib/transfeera/bank_account_validator.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+require 'net/http'
+
+module Transfeera
+    class BankAccountValidator
+
+        def self.validate(catarse_user_bank_account)
+            authorization_data = Transfeera::Authorization.request()
+            account_to_validate = map_to_transfeera_account_from catarse_user_bank_account
+        
+            email_contact = CatarseSettings[:email_contact]
+            access_token = authorization_data['access_token']
+            validation_response = request_validation(email_contact, access_token, account_to_validate)
+            validation = validation_response['_validation']
+            
+            map_to_catarse_bank_account_model validation
+        end
+
+        private
+
+        def self.map_to_transfeera_account_from(catarse_user_bank_account)
+            account_type = if /conta_corrente.*/.match? catarse_user_bank_account.account_type then 'CONTA_CORRENTE' else 'CONTA_POUPANCA' end
```

Comment:
Aqui pra ajudar a encurtar essa linha tu pode identar assim. Aproveitei e troquei o uso de regexp por um `include?` pra facilitar um pouco a leitura do código, e também verifiquei se era conta poupança, só por garantia mesmo. O que tu acha?
```suggestion
            account_type = if catarse_user_bank_account.include?('conta_corrente')
                'CONTA_CORRENTE'
            elsif catarse_user_bank_account.include?('conta_poupanca')
                'CONTA_POUPANCA'
            end
```